Doc/c-api/apiabiversion.rst

```diff
@@ -58,5 +58,13 @@ See :ref:`stable` for a discussion of API and ABI stability across versions.
    Thus ``3.4.1a2`` is hexversion ``0x030401a2`` and ``3.10.0`` is
    hexversion ``0x030a00f0``.
 
+   This version is also available via the symbol :data:`Py_Version`.
+
+.. c:var:: const unsigned long Py_Version
+
+   The Python runtime version number encoded in a single constant integer, with
+   the same format as the c:macro:`PY_VERSION_HEX` macro.
```

Comment:
I'd still like to make it more explicit that the note from the top of the file doesn't apply to `Py_Version`.

```suggestion
   The Python runtime version number encoded in a single constant integer, with
   the same format as the c:macro:`PY_VERSION_HEX` macro.
   This contains the Python version used at run time.
```

Reply:
I agree and that's why I've specified "_runtime_ version number" on line 65. Perhaps adding some _emphasis_ to it would be enough? Otherwise, I'll add the suggested line 67.